Trumpf.Coparoo.Web/Controls/Table/TableDataCell.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace Trumpf.Coparoo.Web.Controls.Table
+{
+    using OpenQA.Selenium;
+    using Trumpf.Coparoo.Web;
+
+    /// <summary>
+    /// Table data cell object.
+    /// </summary>
+    public class TableDataCell : ControlObject
+    {
+        /// <summary>
+        /// Gets the search pattern.
+        /// </summary>
+        protected override By SearchPattern => By.TagName("td");
```

Comment:
I suggest to use an XPath search pattern that matches any cell type. It seems, this can be achieved like in this example: `//div[@class='bubble-title' and contains(text(), 'Cover')]`. More details on [https://sqa.stackexchange.com/a/10416](). Hence, the pattern should match any `td` and the `th`-tag. Then the `TableHeadCell` class becomes obsolete.